Header header
float32 x_max
float32 y_max
float32 z_min
float32 z_max
float32 roll_anlge
